locale/shine/core/frFR.json

```diff
@@ -0,0 +1,64 @@
+{
+	"ADMIN_MENU_COMMANDS_TAB": "Commandes",
+	"ADMIN_MENU_ABOUT_TAB": "À propos de",
+	"ADMIN_MENU_PLAYERS_HELP": "Sélectionnez un joueur (ou des joueurs) et une commande à exécuter.",
+	"NAME": "Nom",
+	"TEAM": "Équipe",
+	"ERROR": "Erreur",
+	"OK": "OK",
+	"ADMIN_MENU_SELECT_SINGLE_PLAYER": "Veuillez sélectionnez un unique joueur",
+	"ADMIN_MENU_OPEN_WIKI": "Ouvrir le Wiki",
+	"ADMIN_MENU": "Admin Menu",
+	"Shuffle": "Aléatoire",
+	"Map Vote": "Map Vote",
+	"Surrender": "Se rendre",
+	"Unstuck": "Se débloquer",
+	"MOTD": "MOTD",
+	"OPEN_IN_STEAM_OVERLAY": "Ouvrir dans l'Overlay Steam",
+	"OPEN_IN_STEAM_OVERLAY_DESCRIPTION": "Ouvrir cette page dans l'overlay Steam ? (Si vous choisissez toujours, tapez \"sh_viewwebinsteam 0\" dans la console pour revenir à cette fenêtre)",
+	"NOW": "Maintenant",
+	"ALWAYS": "Toujours",
+	"PERMANENTLY": "En permanence",
+	"MESSAGE_OF_THE_DAY": "Message du jour",
+	"OPEN_IN_STEAM": "Ouvrir dans Steam",
+	"LOADING": "Chargement...",
+	"MAP": "Carte",
+	"ENABLED": "Activé",
+	"DISABLED": "Désactivé",
+	"ENABLE": "Activer",
+	"DISABLE": "Désactiver",
+	"BACK": "Retour",
+	"REST_OF_MAP": "la durée de la carte en cours.",
+	"ERROR_TAG": "[Erreur]",
+	"TIME_VALUE": "{Value} {TimeUnit}",
+	"TIME_SENTENCE": "{Avant} and {Après}",
```

Comment:
The placeholders here have been translated when they shouldn't be, while ` and ` is left as English.